app/models/concerns/zone/whois_queryable.rb

```diff
@@ -47,7 +47,7 @@ def registrar_vars
   def registrant_vars
```

Comment:
Break each property on a new line here to make it easier to read this code, since there are two properties per line. Thanks